src/pybind/mgr/dashboard/run-frontend-e2e-tests.sh

```diff
@@ -56,22 +56,24 @@ if [ "$BASE_URL" == "" ]; then
     # Set SSL verify to False
     ./bin/ceph dashboard set-rgw-api-ssl-verify False
 
-    BASE_URL=$(./bin/ceph mgr services | jq .dashboard)
+    BASE_URL=$(./bin/ceph mgr services | jq -r .dashboard)
 fi
 
+export BASE_URL
+
 cd $DASH_DIR/frontend
-jq .[].target=$BASE_URL proxy.conf.json.sample > proxy.conf.json
+jq .[].target=\"$BASE_URL\" proxy.conf.json.sample > proxy.conf.json
 
 . $BUILD_DIR/src/pybind/mgr/dashboard/node-env/bin/activate
 timeout 1h npm ci
```

Comment:
Since we are no longer building the frontend, this line can be removed.
```suggestion
```

Reply:
> Since we are no longer building the frontend, this line can be removed.

@tspmelo What do you mean by that?

Reply:
With the changes @alfonsomthd has made, the e2e script no longer builds the frontend.
It now uses the frontend build during make.

Reply:
Done.